templates/_helpers.tpl

```diff
@@ -19,15 +23,19 @@ Create the name of the stackstorm-ha service account to use
 {{- default .Chart.Name .Values.serviceAccount.serviceAccountName -}}
 {{- end -}}
 
-# Generate '-' prefix only when the variable is defined
-{{- define "hyphenPrefix" -}}
+{{/*
+Generate '-' prefix only when the variable is defined
+*/}}
+{{- define "stackstorm-ha.hyphenPrefix" -}}
 {{ if . }}-{{ . }}{{end}}
 {{- end -}}
 
-# Allow calling helpers from nested sub-chart
-# https://stackoverflow.com/a/52024583/4533625
-# https://github.com/helm/helm/issues/4535#issuecomment-477778391
-# Usage: "{{ include "nested" (list . "mongodb" "mongodb.fullname") }}"
+{{/*
+Allow calling helpers from nested sub-chart
+https://stackoverflow.com/a/52024583/4533625
+https://github.com/helm/helm/issues/4535#issuecomment-477778391
+Usage: "{{ include "nested" (list . "mongodb" "mongodb.fullname") }}"
+*/}}
 {{- define "nested" }}
```

Comment:
I didn't prefix this, because I'm not sure if we're overriding a subchart template helper.